ref-exchange/src/errors.rs

```diff
@@ -0,0 +1,12 @@
+// Storage errors //
+
+pub const ERR10_ACC_NOT_REGISTERED: &str = "E20: account not registered";
+pub const ERR11_INSUFFICIENT_STORAGE: &str = "E10: insufficient $NEAR storage deposit";
+pub const ERR12_TOKEN_NOT_WHITELISTED: &str = "E12: token not whitelisted";
+
+// Account Deposits //
+
+pub const ERR21_TOKEN_NOT_REG: &str = "E21: token not registered";
+pub const ERR22_NOT_ENOUGH_TOKENS: &str = "E23: not enough tokens in deposit";
+// pub const ERR23_NOT_ENOUGH_NEAR: &str = "E23: not enough NEAR in deposit";
+pub const ERR24_NON_ZERO_TOKEN_BALANCE: &str = "E24: non-zero token balance";
```

Comment:
For the moment I defined errors only related to storage and account deposits.
If this makes sense I will update other errors as well in other PR.